sphinx/screens/chats/chat-common/chat-common/src/main/java/chat/sphinx/chat_common/ui/viewstate/messageholder/MessageHolderViewState.kt

```diff
@@ -34,6 +36,7 @@ internal val MessageHolderViewState.showSentBubbleArrow: Boolean
 
 internal sealed class MessageHolderViewState(
     val message: Message,
+    @ColorInt val senderColor: Int?,
```

Comment:
because we have access to the Message here, we can instead set the color key's which will move the work from unnecessarily being done upon list creation, to when the data is bound to the holder.